packages/ui/src/theme/css/component/checkbox.scss

```diff
@@ -2,7 +2,7 @@
   cursor: var(--amplify-components-checkbox-cursor);
   align-items: var(--amplify-components-checkbox-align-items);
   gap: inherit;
-  &[data-disabled='true'] {
+  &--disabled {
```

Comment:
Moving this down and combine two selector?

Reply:
+1

Reply:
I've moved the flex property that was below this up but I'm not sure what else can be combined.  There are the Element blocks below this that could be combined under the Block.

Reply:
I mean we have 2 `&--disabled` on the same level? This could be moving down there?